packages/core/ts/__tests__/utils/utils.ts

```diff
@@ -66,8 +66,16 @@ export class TestHarness {
    */
   signup = (user: Keypair): number => this.maciState.signUp(user.pubKey);
 
-  joinPoll = (nullifier: bigint, pubKey: PubKey, newVoiceCreditBalance: bigint, timestamp: bigint): number =>
-    this.poll.joinPoll(nullifier, pubKey, newVoiceCreditBalance, timestamp);
+  /**
+   * Join a poll.
+   *
+   * @param nullifier - The nullifier
+   * @param pubKey - The public key
+   * @param newVoiceCreditBalance - The new voice credit balance
+   * @returns
```

Comment:
missing description of return value

Reply:
No, we don't need to change subgraph because there is no mapping for PollJoined event.

Reply:
ok, maybe we should add it though so that we can track that on the subgraph too (separate to this ofc)